.coafile

```diff
@@ -1,5 +1,35 @@
-[Default]
-bears = SpaceConsistencyBear
-files = **.(html|css)
-ignore = vendors/**
+[default]
+bears = LineCountBear, FilenameBear, InvalidLinkBear, coalaBear
+files = **.md, **.html, **.js, **.css, **.yml
+ignore = **.min.(css|js), _site/**.(html|js|css)
+max_lines_per_file = 600
+
+[html]
+bears = HTMLLintBear, BootLintBear, SpaceConsistencyBear, KeywordBear
+files = **.html
+ignore = _site/**.html, _includes/github.html
+language = HTML
 use_spaces = yeah
+
+[markdown]
+bears = MarkdownBear, SpaceConsistencyBear, KeywordBear
+files = **.md
+language = markdown
+use_spaces = yes
+
+[yaml]
+bears = YAMLLintBear, SpaceConsistencyBear, LineLengthBear, KeywordBear
+files = **.yml
+language = YAML
+use_spaces = yes
+
+[javascript]
+bears = JSComplexityBear
+files = **.js
+ignore = _site/**.js
+
+[css]
+bears = SpaceConsistencyBear
+files = **.css
+ignore = _site/**.css, **.min.css
+use_spaces = yes
```

Comment:
missing newline before EOF, can you please use the bears correctly? Just make a spacing section, run it on all files...